django_filters/filterset.py

```diff
@@ -220,7 +220,7 @@ def __new__(cls, name, bases, attrs):
         # TODO: remove with deprecations
         # check key existence instead of setdefault - prevents unnecessary filter construction
         order_by_field = new_class._meta.order_by_field
-        if opts.order_by and order_by_field not in filters:
+        if opts.order_by and order_by_field not in attrs:
```

Comment:
`declared_filters` contains the parent's generated OrderingFilter however.

So we have to either copy `attrs` for this and/or use another method to mark the automatic OrderingFilter somehow.